GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -682,6 +701,15 @@ + (void)notifyInterceptorsWithMethodSelector:(SEL)methodSelector
   }];
 }
 
+#if UISCENE_SUPPORTED
++ (void)handleSceneWillConnectToNotification:(NSNotification *)notification {
+  if (@available(iOS 13.0, tvOS 13.0, *)) {
+    UIScene *scene = (UIScene *)notification.object;
```

Comment:
I think it will be safer to check the class of the `notification.object`. Even if presence and the class of the object is guaranteed in this iOS version, it may protect us from API changes in the future.

Reply:
Done.